### DISCUSSION TITLE
Ethernet over USB -zperf example  for NUCLEO F429ZI board

### DISCUSSION THREAD

**Discussion by Masood-Salik** [zephyrproject-rtos/zephyr#69056] [General]
Hello everyone,

I need help running the ZPERF example for the NUCLEO F429ZI board with Ethernet over USB. 

https://github.com/zephyrproject-rtos/zephyr/blob/f9739ab41dde9c56e3bc038fe0bebf65b5ac1cfa/samples/net/zperf/sample.yaml#L24-L30

I have configured the proj.conf by doing the above configuration. I also had to disable the CONFIG_NET_IPV6 flag otherwise mcu restart when anything is sent via Serial cmd.

The RNDIS network is discovered and I can set the IP. However, when I run iperf-2.0.5b-win32 in server mode on my Windows 11 and send `zperf tcp upload 192.0.2.2 5001 10 1K 1M` on the board, I get the error 

_**TCP upload failed (-116)**_

. The connection is not established for TCP. Also getting error for UDP upload failed (-115) Can anyone help me understand what the problem might be?

Thank you in advance for your help.

**Comment by erwango** [comment]
@Desvauxm-st Can you have a look ?

**Reply by erwango** [reply]
@Masood-Salik Please consider that @Desvauxm-st will look into this issue when time allows. Also note that there is no issue identified, I don't think this deserve to trigger this whole set of person.
Please understand that all of these guys also have their own concerns and priorities. If they should be involved each time someone struggles to set up a sample, this would quickly not scale and prevent progress on the project as a whole.

**Reply by Masood-Salik** [reply]
@Desvauxm-st Any update?

**Comment by jukkar** [comment]
If you are using picolib then the -116 errno would mean `ETIMEDOUT`, so my guess is that the connection to Windows was not successful.
The issue could be for example wrong IP addresses used between Zephyr and Windows, or various other connection issues.
